rpc.ts

```diff
@@ -20,12 +21,12 @@ export class RPC extends ParseNode {
      * The identifier of the Request object, and whether or not this is
      * streaming.
      */
-    public request: { name: string; streaming?: boolean },
+    public request: { name: Type; streaming?: boolean },
```

Comment:
`name` is a strange name for this property. I suggest changing it to `type`.

Reply:
It seems the `collectTypes` function in testutil.ts has some special handling for keys named "type"...